[2025-05-09T15:04:42.333+0000] {taskinstance.py:1956} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: exercises_pipeline.Exercise-4.run_exercise-4 manual__2025-05-09T14:57:23.120910+00:00 [queued]>
[2025-05-09T15:04:42.345+0000] {taskinstance.py:1956} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: exercises_pipeline.Exercise-4.run_exercise-4 manual__2025-05-09T14:57:23.120910+00:00 [queued]>
[2025-05-09T15:04:42.346+0000] {taskinstance.py:2170} INFO - Starting attempt 1 of 2
[2025-05-09T15:04:42.365+0000] {taskinstance.py:2191} INFO - Executing <Task(BashOperator): Exercise-4.run_exercise-4> on 2025-05-09 14:57:23.120910+00:00
[2025-05-09T15:04:42.373+0000] {standard_task_runner.py:60} INFO - Started process 342 to run task
[2025-05-09T15:04:42.376+0000] {standard_task_runner.py:87} INFO - Running: ['***', 'tasks', 'run', 'exercises_pipeline', 'Exercise-4.run_exercise-4', 'manual__2025-05-09T14:57:23.120910+00:00', '--job-id', '30', '--raw', '--subdir', 'DAGS_FOLDER/exercises_dag.py', '--cfg-path', '/tmp/tmpk4ego5pm']
[2025-05-09T15:04:42.390+0000] {standard_task_runner.py:88} INFO - Job 30: Subtask Exercise-4.run_exercise-4
[2025-05-09T15:04:42.480+0000] {task_command.py:423} INFO - Running <TaskInstance: exercises_pipeline.Exercise-4.run_exercise-4 manual__2025-05-09T14:57:23.120910+00:00 [running]> on host c423529483b0
[2025-05-09T15:04:42.616+0000] {taskinstance.py:2480} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='exercises_pipeline' AIRFLOW_CTX_TASK_ID='Exercise-4.run_exercise-4' AIRFLOW_CTX_EXECUTION_DATE='2025-05-09T14:57:23.120910+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2025-05-09T14:57:23.120910+00:00'
[2025-05-09T15:04:42.618+0000] {subprocess.py:63} INFO - Tmp dir root location: /tmp
[2025-05-09T15:04:42.620+0000] {subprocess.py:75} INFO - Running command: ['/usr/bin/bash', '-c', 'cd /opt/***/app/Exercise-4 && python main.py']
[2025-05-09T15:04:42.650+0000] {subprocess.py:86} INFO - Output:
[2025-05-09T15:04:43.409+0000] {subprocess.py:93} INFO - Found 4 JSON files.
[2025-05-09T15:04:43.410+0000] {subprocess.py:93} INFO - Successfully converted data/file-1.json to data/file-1.csv
[2025-05-09T15:04:43.411+0000] {subprocess.py:93} INFO - Successfully converted data/enough_already/file-4.json to data/enough_already/file-4.csv
[2025-05-09T15:04:43.412+0000] {subprocess.py:93} INFO - Successfully converted data/other_folder/file-3.json to data/other_folder/file-3.csv
[2025-05-09T15:04:43.414+0000] {subprocess.py:93} INFO - Successfully converted data/some_folder/other_folder/file-2.json to data/some_folder/other_folder/file-2.csv
[2025-05-09T15:04:43.415+0000] {subprocess.py:93} INFO - Successfully converted 4 out of 4 files.
[2025-05-09T15:04:43.416+0000] {subprocess.py:97} INFO - Command exited with return code 0
[2025-05-09T15:04:43.447+0000] {taskinstance.py:1138} INFO - Marking task as SUCCESS. dag_id=exercises_pipeline, task_id=Exercise-4.run_exercise-4, execution_date=20250509T145723, start_date=20250509T150442, end_date=20250509T150443
[2025-05-09T15:04:43.487+0000] {local_task_job_runner.py:234} INFO - Task exited with return code 0
[2025-05-09T15:04:43.515+0000] {taskinstance.py:3280} INFO - 1 downstream tasks scheduled from follow-on schedule check
